cmd/bootstrap/README.md

```diff
@@ -41,43 +45,74 @@ unix socket will be used based on the current user and platform selected.
-use `bundle` or `tarball` as the platform.
+## Bootstrap usage
 
-The `bundle` platform will produce a self-extracting shell archive that can
-be executed to install your local non-kubernetes site.
+### Bootstrap command and flags
 
-If you are not comfortable executing the produced script, you can also choose
-`tarball` as the platform. It produces a tar ball that contains a `install.sh`
-script, that basically performs the same procedure of the `bundle`.
-
-Both scripts accept flags:
-
-```
--h               help
--p <platform>    podman, docker, systemd
--x               remove
--d <directory>   dump static tokens
+```shell
+Skupper bootstrap
+
+Bootstraps a nonkube Skupper site base on the provided flags.
+
+When the path (-p) flag is provided, it will be used as the source
+directory containing the Skupper custom resources to be processed,
+generating a local Skupper site using the "default" namespace, unless
+a namespace is set in the custom resources, or if the namespace (-n)
+flag is provided.
+
+A namespace is just a directory in the file system where all site specific
+files are stored, like certificates, configurations, the original sources
+(original custom resources used to bootstrap the nonkube site) and
+the runtime files generated during initialization.
+
+Namespaces are stored under ${XDG_DATA_HOME}/.local/share/skupper/namespaces
+for regular users when XDG_DATA_HOME environment variable is set, or under
+${HOME}/.local/share/skupper/namespaces when it is not set.
+As the root user, namespaces are stored under: /usr/local/share/skupper/namespaces.
+
+In case the path (-p) flag is omitted, Skupper will try to process
+custom resources stored at the sources directory of the default namespace,
+or from the namespace provided through the namespace (-n) flag.
+
+If the respective namespace already exists and you want to bootstrap it
+over, you must provide the force (-f) flag. When you do that, the existing
+Certificate Authorities (CAs) are preserved, so eventual existing incoming
+links should be able to reconnect.
+
+To produce a bundle, instead of rendering a site, the bundle strategy (-b)
+flag must be set to "bundle" or "tarball".
+
+Usage:
+  bootstrap [options...]
+
+Flags:
+  -b string
+    	The bundle strategy to be produced: bundle or tarball
+  -f	Forces to overwrite an existing namespace
+  -n string
+    	The target namespace used for installation
+  -p string
+    	Custom resources location on the file system
+  -v	Report the version of the Skupper bootstrap command
 ```
 
-## Usage
-
 ### Bootstrapping and removing non-kubernetes sites
 
 Place all your Custom Resources (CRs) on a local directory.
 You must always provide a `Site` (CR), plus some other resource
 that makes your running site meaningful, like Listeners and/or Connectors.
 
-In case you want your local site to listen for incoming links from other sites,
+In case you want your local site to accept incoming links from other sites,
```

Comment:
So the RouterAccess CR is only necessary if we want to enable link access in a nonkube site?

Reply:
Ok. New issue for the CLI: #1657